spec/lib/object_serializer_class_methods_spec.rb

```diff
@@ -155,10 +151,70 @@
         end
       end
 
+      after do
+        MovieSerializer.attributes_to_serialize.delete(:title_with_year)
+      end
```

Comment:
I found a small bug in my previous commit. I should not do `MovieSerializer.attributes_to_serialize = {}`. It will mistakenly cleaned up all the attributes from MovieSerializer.